### PR TITLE
correction of Rf.m

### PR DESCRIPTION
**Pull request by gabrielenava**
modified conversion from quaternions to rotation matrix in Rf.m

### CODE REVIEW

File: matlab-src/mex-wholebodymodel_balancing/Rf.m [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
-%calculate the rotation matrix from a quaternion (angle-axis definition)
-
-    theta = u(4);
-    u     = u(1:3); 
-    R     = eye(3) + sin(theta)*Sf(u) + (1 - cos(theta))*Sf(u)*Sf(u);
-
-% qt_b_mod_s = u(1);
-% qt_b_mod_r = u(2:4);
-% R          = eye(3) + 2*qt_b_mod_s*skew(qt_b_mod_r) + 2*skew(qt_b_mod_r)^2;
+%calculate the rotation matrix from a quaternion (real followed by imaginary definition)
```
Comment (traversaro) [inline matlab-src/mex-wholebodymodel_balancing/Rf.m]: Please write the doc in a way that matlab can automatically display it, as in http://it.mathworks.com/help/matlab/matlab_prog/add-help-for-your-program.html . Thanks.

**Comment by gabrielenava** [comment]
what do you mean with "automatically display"? If I type "help Rf" I can already see the documentation. if you mean adding an explanation of the function's structure and not only its description I can do it once for all the mex_wholebodymodel_balancing functions, but I need some more time

**Comment by traversaro** [comment]
@gabrielenava ah, apparently then matlab is able to deal with slightly malformed docstrings... however I thinks it is better to comply with matlab guidelines with consistency, the same that you get when you create a new matlab function:
~~~
function [ output_args ] = untitled( input_args )
%UNTITLED Summary of this function goes here
%   Detailed explanation goes here

end
~~~
In this case this would be:
~~~
function R = Rf(u)
%RF Summary of this function goes here
%   Detailed explanation goes here

end
~~~

**Comment by traversaro** [comment]
@gabrielenava no need to redo all the documentation for all the functions in the repo... let's just start by what we can improve immediately .

**Comment by gabrielenava** [comment]
so the wrong part of the description is "from a given quaternion", because it has been already converted before and what is passed to the function is just  scalar+vector, am I right?

**Comment by gabrielenava** [comment]
I don't really understand what is wrong, to write the description I went on this page 
https://en.wikipedia.org/wiki/Euler%E2%80%93Rodrigues_formula 
 I verify that it's the same formula used in Rf.m than I added a small description based on what is written on this page

**Comment by traversaro** [comment]
@gabrielenava Euler-Rogrigues formula is different from Rodrigues Formula . The first is mapping euler parameters (that are in coordinates identical to quaternions) to rotation matrices, while the second is mapping a rotation expressed in axis angle to rotation matrices.

**Comment by traversaro** [comment]
:+1:

**Comment by gabrielenava** [comment]
I understand and I definitely eliminate the term "Rodriguez formula" to avoid ambiguity

**Comment by DanielePucci** [comment]
Hallelujah!

**Comment by naveenoid** [comment]
:+1:

**Pull request merged**